compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/window/Popup.kt

```diff
@@ -16,14 +16,37 @@
 
 package androidx.compose.ui.window
 
+import androidx.compose.runtime.Composable
 import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Alignment
-import androidx.compose.ui.unit.IntRect
 import androidx.compose.ui.unit.IntOffset
+import androidx.compose.ui.unit.IntRect
 import androidx.compose.ui.unit.IntSize
 import androidx.compose.ui.unit.LayoutDirection
-import androidx.compose.ui.unit.height
-import androidx.compose.ui.unit.width
+
+/**
+ * Properties used to customize the behavior of a [Popup].
+ *
+ * @property focusable Whether the popup is focusable. When true, the popup will receive IME
+ * events and key presses, such as when the back button is pressed.
+ * @property dismissOnBackPress Whether the popup can be dismissed by pressing the back button.
+ * If true, pressing the back button will call onDismissRequest. Note that [focusable] must be
+ * set to true in order to receive key events such as the back button - if the popup is not
+ * focusable then this property does nothing.
+ * @property dismissOnClickOutside Whether the popup can be dismissed by clicking outside the
+ * popup's bounds. If true, clicking outside the popup will call onDismissRequest.
+ */
+@Immutable
+expect class PopupProperties(
```

Comment:
Can we mark it as `@ExperimentalComposeUiApi`?

Reply:
I'm going to mark only constructor as experimental instead of class to avoid opt-in experimental on any usages of Popup function even with default properties argument

Reply:
The idea was to mark this and other functions (Popup itself) as experimental only in commonMain, in case we don't agree when we upstream this in Google.

Is it possible to mark only in commonMain?

Reply:
Done